packages/office-ui-fabric-react/src/components/GroupedList/GroupedListSection.tsx

```diff
@@ -123,12 +124,16 @@ export class GroupedListSection extends BaseComponent<IGroupedListSectionProps,
 
     const { selection, group } = props;
 
+    initializeComponentRef(this);
+
     this._id = getId('GroupedListSection');
 
     this.state = {
       isDropping: false,
       isSelected: selection && group ? selection.isRangeSelected(group.startIndex, group.count) : false
     };
+
+    this._events = new EventGroup(this);
```

Comment:
This one also needs to be disposed

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/c72a0014f9686342bd7acabaabe8fecf0c88af71